internal/lib/sandbox/namespaces_linux.go

```diff
@@ -77,6 +78,11 @@ func pinNamespaces(nsTypes []NSType, cfg *config.Config, idMappings *idtools.IDM
 		"-d", cfg.NamespacesDir,
 		"-f", pinnedNamespace,
 	}
+
+	if len(sysctls) != 0 {
+		pinnsArgs = append(pinnsArgs, "-s", getSysctlForPinns(sysctls))
```

Comment:
Should we worry about cases where we exceed the command line size? If so, should we pass this as temp file like we do for runc exec?

Reply:
```
getconf ARG_MAX
2097152
```
```
sudo sysctl -a |  awk  '{ print $1 }' | wc 
   1884    1884   65129
```
of course, that doesn't account for their actual args, but since its an entire order of magnitude lower, I am not super inclined to worry